taxi/views.py

```diff
@@ -34,6 +35,27 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.edit.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.edit.CreateView):
```

Comment:
The `ManufacturerUpdateView` should inherit from `generic.edit.UpdateView` instead of `generic.edit.CreateView` to properly handle updates.